srcpkgs/mold/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'mold'
+pkgname=mold
+version=1.1
+revision=1
+archs="x86_64*"
+build_style=gnu-makefile
+hostmakedepends="cmake pkg-config"
+makedepends="libstdc++-devel ripgrep openssl-devel zlib-devel"
```

Comment:
Does it really not build without specifying `libstdc++-devel` pkg in `makedepends`?

Reply:
The author explicitly mentions it, I'll test without it.

Reply:
It's important as `libstdc++` brings `std` library for C++, it brings functions like `strdup`, `memcmp` and many others and both `gcc` and `clang` pulls `libstdc++-devel` as dependency.

Reply:
>mold is written in C++20, so if you build mold yourself, you need a recent version of a C++ compiler and a C++ standard library.

From [mold](https://github.com/rui314/mold#how-to-build)'s github

Reply:
if gcc and clang pull them as dependencies, you don't need to add them yourself

Reply:
> if gcc and clang pull them as dependencies, you don't need to add them yourself

Ok, I'll remove it